ministryofjustice/correspondence_tool_staff
# CT-2362 Merge master with offender sars

## Description
Update `master` with latest `offender-sars` branch which includes:

- #794 major refactor of `cases_controller`
- #822 Offender SARs validation
- #782 Offender SARs creation
 
## Self-review checklist

 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2362
 
### Deployment
- Requires database migration to resolve issue of failing migration and to create new Offender SAR correspondence type
 
### Manual testing instructions
- Ensure all case creation and editing journeys work as expected including ICO and Overturned ICOs
- Ensure Offender SAR cannot be created on Production environments
